packages/celotool/src/cmds/deploy/initial/verify-contracts.ts

```diff
@@ -0,0 +1,50 @@
+import * as yargs from 'yargs'
+import { InitialArgv } from '../../deploy/initial'
+import { switchToClusterFromEnv } from 'src/lib/cluster'
+import { getBlockscoutUrl } from 'src/lib/endpoints'
+import { portForwardAnd } from 'src/lib/port_forward'
+import { execCmd } from 'src/lib/utils'
+
+export const command = 'verify-contracts'
+
+export const describe = 'verify the celo smart contracts in blockscout'
+
+export const builder = (argv: yargs.Argv) => {
+  return argv
+    .option('skipClusterSetup', {
+      type: 'boolean',
+      description: 'If you know that you can skip the cluster setup',
+      default: false,
+    })
+    .option('contract', {
```

Comment:
Where is this used?

Reply:
It is not included as a `yarn` task, but you can run for example `bin/celotooljs.sh deploy initial verify-contracts -e integration --contract Attestations --verbose` to only validate the `Attestations` contract. I find quite useful having this option bc some contract may fail to get verified in blockscout and help with debugging without iterating through all of them. 

If you are referring to `skipClusterSetup`, you are right it is not being used. Thanks!